template/config/@control.ts/signals/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@control.ts/signals": "^0.0.4"
```

Comment:
Can we use `latest`?

Reply:
![image](https://github.com/TELEUZI/create-control.ts/assets/81071714/b5bbf6a1-4fa4-4b0e-9599-c4c7bb513008)